packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -744,6 +745,17 @@ const Calendar = Editor.inherit({
     }
   },
 
+  _updateAriaReadonly(readonly = false) {
+    const element = this.$element();
```

Comment:
[naming]
jQuery element should have a dollar prefix
```suggestion
    const $element = this.$element();
```

Reply:
[28bc567](https://github.com/DevExpress/DevExtreme/pull/27898/commits/28bc5675e82786dfe12c0434627ece5fe383cf65)